readme.md

```diff
@@ -32,44 +48,227 @@ Variants will be rendered as `<source>` elements in the order they are declared
   loading='lazy'
   format='webp'
 ></enhance-image>
+=======
+### Installation
+
+Install Enhance Image by running the following command in your Enhance project:
+```shell
+npm install @enhance/image@1.0.0-RC.12
```

Comment:
RC.21 now isn't it?

Reply:
Is sure is but actually this shouldn't mention the RC at all — will fix, thank you!